packages/truffle/test/scenarios/commandRunner.js

```diff
@@ -42,5 +43,36 @@ module.exports = {
         reject(child.error);
       }
     });
-  }
+  },
+  runInDevelopEnvironment: (commands = [], config) => {
+    let child;
+    // open `truffle develop`
+    const cmdLine = "node " +
+      path.join(__dirname, "../../../", "core", "cli.js") +
+      " develop";
```

Comment:
We should follow the same logic as the original commandrunner, using `process.env.NO_BUILD` to pick the correct entry point